kfs-ld/src/main/java/edu/arizona/kfs/module/ld/batch/service/LaborEncumbranceAdjustmentService.java

```diff
@@ -0,0 +1,35 @@
+package edu.arizona.kfs.module.ld.batch.service;
+
+import java.io.File;
+
+public interface LaborEncumbranceAdjustmentService {
+
+    static final String LABOR_PERSONNEL_ENCUMBRANCE_DOC_TYPE = "LLPE";
```

Comment:
LABOR_PERSONNEL_ENCUMBRANCE_DOC_TYPE has reference in serviceImpl where new entries are created. I removed the other one.